.github/renovate.json

```diff
@@ -1,7 +1,17 @@
 {
   "automerge": true,
   "enabledManagers": ["github-actions", "npm"],
-  "ignoreDeps": ["cypress"],
+  "ignoreDeps": [
+    "@nrwl/cli",
+    "@nrwl/devkit",
+    "@nrwl/jest",
+    "@nrwl/nx-cloud",
+    "@nrwl/tao",
+    "@types/jest",
+    "cypress",
+    "jest",
+    "ts-jest"
```

Comment:
how come nx manages these deps for us?

Reply:
We don't have to let it, but Nx comes with an executor for jest and so if we leverage that we can basically forget about jest upgrades and no that the Nx Team will test the collection of packages e2e on each release...

If we want to stay on the bleeding edge of jest at all times then we choose not to follow that approach.

WDYT?

Reply:
Same goes for cypress

Reply:
I kept jest and cypress out of the list for now